web/tests/e2e/helpers/hasura/admin-graphql-client.ts

```diff
@@ -0,0 +1,10 @@
+import { GraphQLClient } from "graphql-request";
+
+export const adminGraphqlClient = new GraphQLClient(
+  process.env.NEXT_PUBLIC_GRAPHQL_API_URL!,
+  {
+    headers: {
+      "x-hasura-admin-secret": process.env.HASURA_GRAPHQL_ADMIN_SECRET!,
```

Comment:
Where will we be getting this admin secret from? Also this function will not be callable? Outside of tests?

Reply:
Can you share how this will be set up? Will you trigger a docker image at start and set the env.test value? otherwise I don't see how admin HASURA_GRAPHQL_ADMIN_SECRET is being set. It's not defined in env.test and probably shouldn't be until runtime if you are creating a new instance and DB on the fly